bokeh/models/widgets/tables.py

```diff
@@ -1,740 +1,764 @@
+    -0.23        '.00'          -.23
+    -0.23        '(.00)'        (.23)
+    0.23         '0.00000'      0.23000
+    0.23         '0.0[0000]'    0.23
+    1230974      '0.0a'         1.2m
+    1460         '0 a'          1 k
+    -104000      '0a'           -104k
+    1            '0o'           1st
+    52           '0o'           52nd
+    23           '0o'           23rd
+    100          '0o'           100th
+    ============ ============== ===============
+
+    **CURRENCY**:
+
+    =========== =============== =============
+    Number      Format          String
+    =========== =============== =============
+    1000.234    '$0,0.00'       $1,000.23
+    1000.2      '0,0[.]00 $'    1,000.20 $
+    1001        '$ 0,0[.]00'    $ 1,001
+    -1000.234   '($0,0)'        ($1,000)
+    -1000.234   '$0.00'         -$1000.23
+    1230974     '($ 0.00 a)'    $ 1.23 m
+    =========== =============== =============
+
+    **BYTES**:
+
+    =============== =========== ============
+    Number          Format      String
+    =============== =========== ============
+    100             '0b'        100B
+    2048            '0 b'       2 KB
+    7884486213      '0.0b'      7.3GB
+    3467479682787   '0.000 b'   3.154 TB
+    =============== =========== ============
+
+    **PERCENTAGES**:
+
+    ============= ============= ===========
+    Number        Format        String
+    ============= ============= ===========
+    1             '0%'          100%
+    0.974878234   '0.000%'      97.488%
+    -0.43         '0 %'         -43 %
+    0.43          '(0.000 %)'   43.000 %
+    ============= ============= ===========
+
+    **TIME**:
+
+    ============ ============== ============
+    Number       Format         String
+    ============ ============== ============
+    25           '00:00:00'     0:00:25
+    238          '00:00:00'     0:03:58
+    63846        '00:00:00'     17:44:06
+    ============ ============== ============
+
+    For the complete specification, see http://numbrojs.com/format.html
+    """)
+
+    language = Enum(NumeralLanguage, default="en", help="""
+    The language to use for formatting language-specific features (e.g. thousands separator).
+    """)
+
+    rounding = Enum(RoundingFunction, help="""
+    Rounding functions (round, floor, ceil) and their synonyms (nearest, rounddown, roundup).
+    """)
+
+class BooleanFormatter(CellFormatter):
+    ''' Boolean (check mark) cell formatter.
+
+    '''
+
+    icon = Enum('check', 'check-circle', 'check-circle-o', 'check-square', 'check-square-o', help="""
+    The icon visualizing the check mark.
+    """)
+
+class CompositeFormatter(CellFormatter):
+    '''CompositeFormatter applies multiple formatters to a single TableColumn
+
+    Example:
+    .. code-block:: python
+
+        example_formatter = CompositeFormatter(formatters=[StringFormatter(font_style="bold"), HTMLTemplateFormatter(
+        template='<code><%= value %></code>')])

```

Comment:
nit: It's common to indent a line continuation, and we definitely adhere to that style throughout. To be consistent I would suggest something like:
```
        example_formatter = CompositeFormatter(formatters=[
            StringFormatter(font_style="bold"), 
            HTMLTemplateFormatter(template='<code><%= value %></code>')
        ])
```